### PR TITLE
add jclasslib-bytecode-viewer v5.1

### PR DESCRIPTION
**Pull request by diguage**
If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.

After making all changes to the cask:

- [x] brew cask audit --download {{cask_file}} is error-free.
- [x] brew cask style --fix {{cask_file}} reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask:**

- [x] Named the cask according to the token reference.
- [x] brew cask install {{cask_file}} worked successfully.
- [x] brew cask uninstall {{cask_file}} worked successfully.
- [x] Checked there are no open pull requests for the same cask.
- [x] Checked the cask was not already refused in closed issues.
- [x] Checked the cask is submitted to the correct repo.

### CODE REVIEW

File: Casks/jclasslib-bytecode-viewer.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'jclasslib-bytecode-viewer' do
+  version '5.1'
+  sha256 '02231f57c3672851ea3ce8d9a304641a3c278e4e71abb73067f9db0c6e5f5d64'
+
+  url "https://github.com/ingokegel/jclasslib/releases/download/#{version}/jclasslib_macos_#{version.dots_to_underscores}.dmg"
+  name 'jclasslib bytecode viewer'
```
Comment (vitorgalvao) [inline Casks/jclasslib-bytecode-viewer.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/jclasslib-bytecode-viewer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  homepage 'https://github.com/ingokegel/jclasslib'
+
+  auto_updates true
+
+  app "jclasslib bytecode viewer.app"
```
Comment (miccal) [inline Casks/jclasslib-bytecode-viewer.rb]: Please use single-quotes here.

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/jclasslib-bytecode-viewer.rb (hunk 2 above)
Comment (diguage) [inline Casks/jclasslib-bytecode-viewer.rb]: Ok, I will correct the punctuation. Thanks.

File: Casks/jclasslib-bytecode-viewer.rb (hunk 1 above)
Comment (diguage) [inline Casks/jclasslib-bytecode-viewer.rb]: Ok, I will add. Thanks.

**Comment by diguage** [comment]
I have modified the code as suggested.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**